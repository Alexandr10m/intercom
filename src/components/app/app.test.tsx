import * as React from "react";
import * as render from "react-test-renderer";
import App from "./app";

describe(`Snapshot of App`, () => {
  it(`AppComponent shoul render h1`, () => {
    const tree = render
      .create(<App/>)
      .toJSON();

      expect(tree).toMatchSnapshot();
  });
});
