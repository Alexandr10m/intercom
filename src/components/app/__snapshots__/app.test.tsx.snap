// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of App AppComponent shoul render h1 1`] = `
<h1>
   HI
</h1>
`;
